### ISSUE TITLE
Running carto-vl in React

### ISSUE THREAD

**Issue by rafaelfernandezgis** [CartoDB/carto-vl#939]
HI there,
I'm trying to run carto-vl in React. For this, I put carto-vl as a dependency in my package.json project:
```
"dependencies": {
    "@carto/carto-vl": "0.7.0",
    "mapbox-gl": "0.48.0",
    "react": "^16.4.2",
    "react-dom": "^16.4.2",
    "react-scripts": "1.1.5",
    "react-mapbox-gl": "3.9.2"
  },
```
and then I imported it as usual:
```
import Carto from '@carto/carto-vl'
```
All what I get is `undefined` for the Carto object. So I can't use it.

Is actually carto-vl ready for React?

**Comment by elenatorro** [comment]
Hi @rafaelfernandezgis , let me check our package in npm. We recently add the option to import CARTO VL as you show in your example in this PR https://github.com/CartoDB/carto-vl/pull/902. I will contact you as soon as possible. Thank you!

**Comment by rafaelfernandezgis** [comment]
Thanks a lot @elenatorro

**Comment by elenatorro** [comment]
Hi again @rafaelfernandezgis, we just released the `0.8.0` version, so by doing `import carto from '@carto/carto-vl'` (`carto` lowercase) should work fine. If it does, please tell us so we can close the issue. Thank you very much!

**Comment by rafaelfernandezgis** [comment]
It works now using `import carto from '@carto/carto-vl'`
👍 

Thx!
